frontend/app/components/auth/__anonymous-login-form/auth__anonymous-login-form.tsx

```diff
@@ -26,7 +28,7 @@ export const messages = defineMessages({
   },
   symbolLimit: {
     id: 'anonymousLoginForm.symbol-limit',
-    defaultMessage: 'Username must start with a letter and contain only latin letters, numbers, underscores, or spaces',
```

Comment:
did you drop "and contain" by accident?

Reply:
Fixed. But I am not sure that it is right message.